src/AutomationTests/SystemUnitTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Abstractions;
+using Axe.Windows.Concretions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Axe.Windows.AutomationTests
+{
+    using Path = System.IO.Path;
+
+    [TestClass]
+    public class SystemUnitTests
+    {
+        private readonly ISystem _system = SystemFactory.CreateSystem();
+
+        [TestMethod]
+        [Timeout(1000)]
+        public void DateTime_Matches()
+        {
+            var now1 = _system.DateTime.Now;
+            var now2 = System.DateTime.Now;
+
+            var timeSpan = now2.Subtract(now1);
+
+            Assert.IsTrue(timeSpan.TotalSeconds <= 1);
+        }
+
+        [TestMethod]
+        [Timeout(1000)]
+        public void CurrentDirectory_Matches()
+        {
+            Assert.AreEqual(System.Environment.CurrentDirectory, _system.Environment.CurrentDirectory);
```

Comment:
>CurrentDirectory [](start = 47, length = 16)

This test feels like it's testing both System and SystemIO classes. Should they be split?